### PR TITLE
OSDOCS-1017 Release Notes 4.5 - Storage

### PR DESCRIPTION
**Pull request by bobfuru**
[OSDOCS-1017](https://issues.redhat.com/browse/OSDOCS-1017)
Adds 4.5 OCP Storage features to 4.5 Release Notes:
• CSI AWS EBS Operator (TP)
• CSI Inline Ephemeral Volumes (TP)
• CSI Manila Operator (GA)
• CSI Cloning GA

Also updates the TP tracker table

Preview build: http://file.rdu.redhat.com/bfuru/061120/OSDOCS-1017/release_notes/ocp-4-5-release-notes.html

### CODE REVIEW

File: release_notes/ocp-4-5-release-notes.adoc [hunk 1]
```diff
@@ -327,7 +354,7 @@ In the table below, features are marked with the following statuses:
 |Ephemeral Storage Limit/Requests
 |TP
 |TP
-|
+|TP
```
Comment (bobfuru) [inline release_notes/ocp-4-5-release-notes.adoc:357]: @jsafrane Is this feature still in Tech Preview? I'm not familiar with its origin. It seems to have started in this table with 4.1. In 3.11, we had this: https://docs.openshift.com/container-platform/3.11/release_notes/ocp_3_11_release_notes.html#ocp-311-local-ephemeral-storage
Not sure if they're related, or if we even need this entry in the tracker anymore?

**Comment by bobfuru** [comment]
@jsafrane and @qinpingli PTAL - links to docs for AWS EBS and Manila CSI Operators will be added in their related Release Notes entries when those Operator PRs are approved/merged for 4.5. Thanks!

**Comment by bobfuru** [comment]
@openshift/team-documentation PTAL

**Comment by bmcelvee** [comment]
LGTM

File: release_notes/ocp-4-5-release-notes.adoc (hunk 1 above)
Comment (jsafrane) [reply release_notes/ocp-4-5-release-notes.adoc:357]: Yes, it's still beta upstream.

**Comment by qinpingli** [comment]
@bobfuru For `External provisioner for AWS EFS` feature, I can confirm this feature is dropped from 4.4. But there is a user story in 4.2, so I don't know if we should move this feature to "Table 1".

**Comment by bobfuru** [comment]
> @bobfuru For `External provisioner for AWS EFS` feature, I can confirm this feature is dropped from 4.4. But there is a user story in 4.2, so I don't know if we should move this feature to "Table 1".

@qinpingli A few questions about this:

- By "dropped", do you mean "removed" or "deprecated"?
- Which user story are you referring to in 4.2?
- Should we also move this feature to Table 1 in 4.3?

Also, step #1 in the [Create the EFS Provisioner](https://docs.openshift.com/container-platform/4.2/storage/persistent_storage/persistent-storage-efs.html#efs-provisioner_persistent-storage-efs) procedure includes the `quay.io/external_storage/efs-provisioner:latest` container image. Is this still correct?

**Comment by qinpingli** [comment]
> > @bobfuru For `External provisioner for AWS EFS` feature, I can confirm this feature is dropped from 4.4. But there is a user story in 4.2, so I don't know if we should move this feature to "Table 1".
> 
> @qinpingli By "dropped", do you mean "deprecated" (not "removed")? Also, which user story are you referring to in 4.2?
> 
> Based on your suggestion, I moved it from Table 2 to Table 1, indicating that it was still TP in 4.3 but deprecated in 4.4 and 4.5. If that is correct, we should probably also update 4.4 Release Notes.
> 
> Also, step 1 in the [Persistent storage using AWS EFS -> Create the EFS provisioner](https://docs.openshift.com/container-platform/4.2/storage/persistent_storage/persistent-storage-efs.html#efs-provisioner_persistent-storage-efs) procedure includes the `quay.io/external_storage/efs-provisioner:latest` container image in the spec. Based on this change for EFS external provisioner, is this sample still correct?

@bobfuru 
This is another PR to follow this. https://github.com/openshift/openshift-docs/pull/22524#issuecomment-636909765

user story about efs: https://issues.redhat.com/browse/OSDOCS-306

**Comment by qinpingli** [comment]
> @qinpingli Based on your [comment](https://github.com/openshift/openshift-docs/pull/22524#issuecomment-637233287) in PR 22524, it sounds to me like EFS should be `DEP` in 4.2 and 4.3 because it wasn't tested, and `-` (Removed) in 4.4 and 4.5 because the image is not available for customers.
> Two questions for you:
> 
> 1. Is this a correct assessment? If so, I'll work with Cody to accurately reflect this info in 4.2+ Release Notes tables.
Thanks, Bob. the assessment lgtm.
> 2. What does this mean for the 4.x EFS documentation (https://docs.openshift.com/container-platform/4.4/storage/persistent_storage/persistent-storage-efs.html)? Should all of it be removed starting from 4.4?
I agree with remove it starting from 4.4.

**Comment by bobfuru** [comment]
@qinpingli Based on feedback from PM/Dev (https://github.com/openshift/openshift-docs/pull/22524#issuecomment-642721496), I'm going to leave the docs in 4.4 and @codyhoag will keep EFS as TP in the Release Notes for 4.2+. We can revisit when EFS is included in a CSI driver.

**Comment by jsafrane** [comment]
@bobfuru lgtm

**Pull request merged**